test/Test.hs

```diff
@@ -104,7 +104,7 @@ main = hspec $ do
 
       it "don't throw when no time out" $ runSessionWithConfig (def {messageTimeout = 5}) "hie" fullCaps "test/data/renamePass" $ do
         loggingNotification
-        liftIO $ threadDelay 10
+        liftIO $ threadDelay $ 10 * 1000000
```

Comment:
@bubba Not sure it it was done intencionally, but test did not terminate in my windows 7 with 10

Reply:
Yes, I presume I meant to write 10s not 10us!